### DISCUSSION TITLE
GNR /SRF support of this tool

### DISCUSSION THREAD

**Discussion by ugpatel** [opcm/pcm#551] [General]
Hi , when this tool can be supported for GNR and SRF ? also what is a process if we want to request this tool for future generations of server platforms like CWF ?

**Comment by ogbrugge** [comment]
Please contact us directly on our intel email addresses using your own
intel email address, if you are a customer talk to your intel contact and
have him talk to us.

**Reply by ugpatel** [reply]
which Intel address to use ?

**Comment by ogbrugge** [comment]
Look up my name or Roman's (his last name is in the source code) name in
the intel address book, and if you do not have access to it then you will
have to contact your intel contact.
